build.gradle

```diff
@@ -492,14 +492,9 @@ subprojects {
         }
 
         plugins.withId("java") {
-            task javadocJar(type: Jar) {
-                classifier = 'javadoc'
-                from javadoc
-            }
-
-            task sourcesJar(type: Jar) {
-                classifier = 'sources'
-                from sourceSets.main.allSource
```

Comment:
The built-in will also pick source from the main source set only?

Reply:
I mean the proto codegen will not be picked.

Reply:
Yes, it produces the same results. Honestly, it would be better to include the generated code in my mind, but it's not that big of a deal.